lightning-tag/data-collection.md

```diff
@@ -75,9 +63,11 @@ window.lotame_<lotameClientId>.collect({
 });
 ```
 
-### New Page Load Events
+This option takes the same `{data}` object as described in [Lightning Tag Data Collection](lightning-tag/detailed-reference?id=data-object).
+
+## Triggering New Page Load Events
 
-Lightning Tag provides a `page()` method that you can pass data on just like the `collect()` method described above. Unlike the `collect()` method, the `page()` method records a pageView in Lotame and initiates targeting again which will update audiences in your audience Callback or LocalStorage. 
+Lightning Tag provides a `page()` method that you can pass data in on just like the `collect()` method described above. Unlike the `collect()` method, the `page()` method records a page view in Lotame and initiates targeting again which will update audiences in your audience callback or local storage as well as fire any additional sync pixels, export beacons, or automated data collection rules for the page. 
```

Comment:
How about this instead?:

Lightning Tag provides a page() method that you can pass data in on just like the collect() method described above. Unlike the collect() method, the page() method records a page view in Lotame and reinitiates all Lightning Tag functions that normally occur on pageload. This includes data collection, targeting (updates audiences in your audience callback or local storage), as well as firing any applicable 3rd party pixels (sync pixels & export beacons)

Reply:
updated